src/components/MewOverlay/MewOverlay.vue

```diff
@@ -59,35 +65,59 @@
         fluid
       >
         <!--
-  =====================================================================================
-    White sheet (displays on the overlay - size is based on the contentSize prop)
-  =====================================================================================
-  -->
+        =====================================================================================
+          White sheet (displays on the overlay - size is based on the contentSize prop)
+        =====================================================================================
+        -->
         <v-row class="ma-0 pa-0 d-flex align-center justify-center flex-column">
           <v-sheet
             :width="isMobile ? '100%' : sheetWidth"
             height="100%"
             color="white"
-            :class="['white-sheet-container', isMobile ? 'mt-0' : 'mt-4']"
+            class="pt-5"
```

Comment:
can you add to the class below please so it looks: 
`            :class="[
              !isMobile ? 'white-sheet-container' : '',
              isMobile ? 'my-0' : 'my-4',
              'pt-5'
            ]"`

Reply:
I fixed it by adding  pt-md-0.